src/genlib_rational.erl

```diff
@@ -2,9 +2,9 @@
 %%% Genlib
 %%% Rational number arithmetic subset
 %%%
-%%% # Rounding rule
+%%% # Rounding methods
 %%%
-%%% Commercial rounding [1], round up iff the remainder is not less than the
+%%% round_half_away_from_zero - round up if the remainder is not less than the
```

Comment:
Can we keep markdown style here? Maybe something like that:

```
## [round_half_away_from_zero][1]

...

[1]: ...
```

Moreover, that dangling reference-style link will be used properly.